### PR TITLE
Npm

### PR DESCRIPTION
**Pull request by trusktr**

### CODE REVIEW

**Comment by guybedford** [comment]
These don't need to be on npm.

**Comment by trusktr** [comment]
Why not? By putting them on npm they have 'official' names. They have an official, unambiguous, definitive origin, since npmjs is essentially the definitive source for anything that's a JavaScript package.

By naming the system css plugin as "css" in the jspm registry, you're claiming a name that doesn't really fit, and doesn't work with the scheme you've set up regarding the registry properties.

Basically, the plugin just needs to work, and a registry property shouldn't be a barrier to this. I don't see why this barrier needs to exist.

**Comment by guybedford** [comment]
@trusktr I know it's frustrating if you're excited about the project but hitting snags. Have patience, I'm doing my best.
